cardano-ping/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Revision history for cardano-ping
 
+## 0.2.0.0 -- 2023-05-08
+
+* Support for `NodeToNodeV_15` and `NodeToClientV_11`, e.g. support for
```

Comment:
This should be NodeToNodeV_11 and NodeToClientV_15, you swapped the numbers :)

Reply:
Fixed.